ui/arduino2/store.js

```diff
@@ -552,6 +566,13 @@ async function store(state, emitter) {
     emitter.emit('render')
   })
   emitter.on('download-files', async () => {
+    filesNameList = state.selectedFiles.map(f => f.fileName)
```

Comment:
This present the same issues mentioned before:
- Variables not initialized properly
- Unnecessary `console.log`
- if/else logic that leaves margin for javascript error
- Missing `log` statement (this is more of an improvement)